swift/ql/test/extractor-tests/statements/ConditionElements.expected

```diff
@@ -1,5 +1,5 @@
 | main.swift:3:8:3:13 | ConditionElement | main.swift:3:8:3:13 | BinaryExpr |
-| main.swift:10:17:10:24 | ConditionElement | main.swift:10:17:10:24 | ParenExpr |
+| main.swift:10:17:10:24 | ConditionElement | main.swift:10:18:10:22 | BinaryExpr |
```

Comment:
This change happens because of the implementation of `convertsFrom` in [`IdentityExpr`](https://github.com/github/codeql/blob/2882c42698aea8f6f627455e8e526fd8e10f4930/swift/ql/lib/codeql/swift/elements/expr/IdentityExpr.qll#L4). So now we skip past the parentheses.